<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OfficersForm</class>
 <widget class="QWidget" name="OfficersForm">
  <property name="windowTitle">
   <string>Officers</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="titleLabel">
     <property name="text">
      <string>Officers</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
     <property name="styleSheet">
      <string notr="true">font-weight:600; font-size:14pt;</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">

     <item row="0" column="0">
      <widget class="QLabel" name="label_0">
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
<widget class="QSpinBox" name="spinId">
       <property name="maximum"><number>2147483647</number></property>
       <property name="readOnly"><bool>true</bool></property>
      </widget>
     </item>

     <item row="1" column="0">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>Official Reg Number</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
<widget class="QLineEdit" name="editOfficialRegNumber">
       <property name="placeholderText"><string>NRP/NRK</string></property>
      </widget>
     </item>

     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
<widget class="QLineEdit" name="editName">
       <property name="placeholderText"><string>Nama lengkap</string></property>
      </widget>
     </item>

     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Photo (path/url)</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
<widget class="QLineEdit" name="editPhoto">
       <property name="placeholderText"><string>opsional</string></property>
      </widget>
     </item>

     <item row="4" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Email</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
<widget class="QLineEdit" name="editEmail">
       <property name="placeholderText"><string>email@contoh.com</string></property>
      </widget>
     </item>

     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
<widget class="QLineEdit" name="editPassword">
       <property name="echoMode"><enum>QLineEdit::Password</enum></property>
      </widget>
     </item>

     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Segment</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
<widget class="QLineEdit" name="editSegment">
       <property name="placeholderText"><string>Divisi/Sektor</string></property>
      </widget>
     </item>

     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>License Expired</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
<widget class="QDateEdit" name="dateLicenseExpired">
       <property name="calendarPopup"><bool>true</bool></property>
      </widget>
     </item>

     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>OneSignal</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
<widget class="QLineEdit" name="editOneSignal">
       <property name="placeholderText"><string>Player ID (opsional)</string></property>
      </widget>
     </item>

     <item row="9" column="0">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Created At</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
<widget class="QLineEdit" name="editCreatedAt">
       <property name="placeholderText"><string>otomatis (read-only)</string></property>
      </widget>
     </item>

     <item row="10" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Updated At</string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
<widget class="QLineEdit" name="editUpdatedAt">
       <property name="placeholderText"><string>otomatis (read-only)</string></property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item><widget class="QPushButton" name="btnNew"><property name="text"><string>Baru</string></property></widget></item>
     <item><widget class="QPushButton" name="btnSave"><property name="text"><string>Simpan</string></property></widget></item>
     <item><widget class="QPushButton" name="btnUpdate"><property name="text"><string>Ubah</string></property></widget></item>
     <item><widget class="QPushButton" name="btnDelete"><property name="text"><string>Hapus</string></property></widget></item>
     <item><widget class="QPushButton" name="btnClear"><property name="text"><string>Bersihkan</string></property></widget></item>
     <item><spacer name="hsp"><property name="orientation"><enum>Qt::Horizontal</enum></property>
      <property name="sizeHint" stdset="0"><size><width>40</width><height>20</height></size></property></spacer></item>
     <item><widget class="QLineEdit" name="lineSearch"><property name="placeholderText"><string>Cari...</string></property></widget></item>
     <item><widget class="QPushButton" name="btnRefresh"><property name="text"><string>Refresh</string></property></widget></item>
    </layout>
   </item>

    <item>
     <widget class="QTableView" name="tableView">
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
